### ISSUE TITLE
spreadsheet to json

### ISSUE THREAD

**Issue by subinkk08** [joelparkerhenderson/demo_swift_excel_xlsx_reader_writer#1]
Is it possible to convert to json data ?.
if not ,I need to get all row values of sheet. how to get that ?please help

**Comment by joelparkerhenderson** [comment]
Yes. This demo is focusing on how to extract data from Excel to text format. 

You can convert the text to JSON. For examples, you may want to look at SwiftyJSON and Alamofire.

If you are doing work for a company project, or want to pay for me to write what you need, please let me know and I can provide a cost/time estimate for you.

Thank you!

**Comment by subinkk08** [comment]
good day,
Thanks for the reply. i got all cell values from the sheet but i dont know how to get all values based on a row. so that i can loop it and save to a object directly.is there any direct method from worksheet property ?.

**Comment by joelparkerhenderson** [comment]
Yes. You can use `BRARow` and rows.cells. See the excellent Objective C code in this repo, and in particular this file, and the BRARow example:

https://github.com/renebigot/XlsxReaderWriter/blob/master/XlsxReaderWriter/BRAWorksheet.m
